#!/usr/bin/env python
PACKAGE = "hektar"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()
gen.add("ticks_to_stop", int_t, 0, "number of ticks until robot comes to a stop", 0, 0, 72)

exit(gen.generate(PACKAGE, "hektar", "DeadReckon"))

